evm/deploy-contract.js

```diff
@@ -26,8 +26,47 @@ const {
     prompt,
     mainProcessor,
     isContract,
+    addDeploymentOptions,
 } = require('./utils');
 
+function getContractPath(contractName) {
+    switch (contractName) {
+        case 'AxelarServiceGovernance': {
+            return '@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/governance/AxelarServiceGovernance.sol/AxelarServiceGovernance.json';
+        }
+
+        case 'InterchainProposalSender': {
+            throw new Error(`Artifact path for ${contractName} must be entered manually.`);
```

Comment:
Please confirm that this artifact path needs to be entered manually

Reply:
yeah, we don't depend on this repo atm